spec/config/secrets.defaults.yml

```diff
@@ -0,0 +1,6 @@
+---
+openstack:
+  hostname: 11.22.33.44
+  port: '5000'
```

Comment:
NOTE to self I'm surprised port is explicitly a string here seems like it'd be easier to let it be an int, I'll follow-up